composer.json

```diff
@@ -13,17 +13,14 @@
         "akeneo/measure-bundle": ">=0.5.0"
```

Comment:
@fitn @jmleroux : Since the MeasureBundle was reintegrated in the PIM from v1.6 (the oldest supported version), maybe we should require akeneo/pim-community-dev instead?
The current problem is, when you composer install with a PHP v7.1 from, the current configuration has you download symfony/symfony v4, although the PIM is based on v3.4

Reply:
Yep you can require akeneo/pim-community-dev. It seems better as it is our real use case.